addons/website/static/src/systray_items/edit_in_backend.js

```diff
@@ -3,14 +3,16 @@
 import { registry } from "@web/core/registry";
 import { useService, useBus } from "@web/core/utils/hooks";
 
-const { Component, onWillStart, onMounted } = owl;
+const { Component, onWillStart, onMounted, useState } = owl;
 
 const websiteSystrayRegistry = registry.category('website_systray');
 
 export class EditInBackendSystray extends Component {
     setup() {
         this.websiteService = useService('website');
         this.actionService = useService('action');
+        this.defaultObjectName = this.env._t('Data');
+        this.state = useState({mainObjectName: this.defaultObjectName});
```

Comment:
You don't need to set up a default, it will be initialized by `willStart` and never rendered before. The value can be the emply string or probably any falsy value meanwhile.